websites/P/Paramount+/metadata.json

```
@@ -13,7 +13,7 @@
 		"www.paramountplus.com",
 		"paramountplus.com"
 	],
-	"version": "1.0.14",
+	"version": "2.0.0",
 	"logo": "https://i.imgur.com/VWrl85V.jpg",
```

```suggestion
	"logo": "https://i.imgur.com/4Z2Dwvc.jpg",
```
Please update it here too